src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -8187,5 +8187,294 @@ .maxstack  3
 }
 ", sequencePoints: "Program+<Test>d__0.MoveNext", source: source);
         }
+
+        [Fact]
+        public void ExpressionVariable_InFieldInitializer()
```

Comment:
>ExpressionVariable_InFieldInitializer [](start = 20, length = 37)

This change is likely to be affected by the change in [https://github.com/dotnet/roslyn/pull/25441](https://github.com/dotnet/roslyn/pull/25441). Also, that PR probably covers all the scenarios covered by the new tests here.